.github/workflows/go.yml

```diff
@@ -0,0 +1,82 @@
+---
+
+name: Go
+
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+
+jobs:
+
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+
+      - uses: Jerome1337/golint-action@v1.0.2
+        with:
+          golint-path: '.'
+
+  vet:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+
+      - uses: actions/setup-go@v2
+        with:
+          stable: false
+          go-version: '^1'
+
+      - run: go vet
```

Comment:
Can we use `go vet ./...`, like we want for the lint?

Reply:
> I had this set to `./...` originally, which is also my preference, however this picks up on the [nishanths/go-hgconfig](https://github.com/nishanths/go-hgconfig) and [tcnksm/go-gitconfig](https://github.com/tcnksm/go-gitconfig) modules in the `vendors/` directory. Since you (@nishanths) are the author of `go-hgconfig` we can easily make sure that that module passes the linter, but to make `go-gitconfig` pass we either need to make local changes that divert from the upstream, or fix the issues by opening PRs to the upstream directly.
> 
> Since this is your repository, I'll leave that call up to you. 👍

Same as above.

Reply:
Same, we can keep the current behavior.